dts/arm/nxp/nxp_lpc55S6x.dtsi

```diff
@@ -12,7 +12,7 @@
 / {
 	soc {
 		sram: sram@14000000 {
-			ranges = <0x400000 0x1400000 0x20000000>;
+			ranges = <0x4000000 0x14000000 0x20000000>;
```

Comment:
Would it make more sense to set the size to 0x1c044000? I'm not sure the larger address hurts anything, but it does technically push the address into the next region of the address space.

Reply:
I left it bigger to catch any future variants of the lpc55s6x that might have more SRAM.

Reply:
Perhaps at least reduce it to 0x20000000-0x04000000. Right now, it extends into the next address space, which are device control registers.

Reply:
Which, if my math is correct would be a size of 0x1c000000.

Reply:
In Non-secure it goes to 0x24000000 in secure it goes to 0x34000000.  Seems like the next address space starts at 0x40000000 non-secure and 0x50000000 secure.